internal/fetch/fetch.go

```diff
@@ -56,11 +56,16 @@ type Repo struct {
 
 	// Repo is the name of the repository, such as `googleapis` or `google-cloud-rust`.
 	Repo string
+
+	// Branch is the name of the repository branch, such as `master` or `preview`.
+	Branch string
```

Comment:
nit: order alphabetically

Reply:
For sure. Done where I've made changes. Filed a bug to fix other places.